Code Review Lab

abemille opened the pull request.

Commit:
Initial commit

abemille commented:
Get rid of print statement in countOccurrences().  Add unit tests.  Add comments.

venigallav commented on Exercise1.java:
```diff
@@ -0,0 +1,27 @@
+     */
+    public static void main(String[] args) {
+
+        Path textFile = Paths.get("xanadu.txt");
```
Its better to assign file name to a final String and use it as a parameter

Commit:
final added final modifier.

Added final modifier to file name in Exercise1.java.  Note that print
statement was removed from countOccurrences() prior to initial commit.
By adding final, I have made the two changes for this homework.

abemille commented:
Added final modifier to path/file name in Exercise1.java.  Note that print statement was removed from countOccurrences() prior to initial commit.  (Since this was a simple change, I took care of this during our code review.)  By adding final and removing the print statement from countOccurrences(), I have made the two changes required for this homework.

gingeraltoids commented:
This has been reviewed.  Good job.
